src/Proxy.hs

```diff
@@ -830,9 +830,8 @@ getTransactionCostR = withExchangeRate $ \(rate, pv) -> do
         consensusInfoRes <- getConsensusInfo
         case getResponseValueAndHeaders consensusInfoRes of
             Left errRes -> return errRes
-            Right (csRes, hds) -> do
-              best <- getBlockHashHeader hds
-              chainParamsRes <- getBlockChainParameters (Given best)
+            Right (csRes, _) -> do
+              chainParamsRes <- getBlockChainParameters Best
```

Comment:
Was this introduced with the following?

https://github.com/Concordium/concordium-wallet-proxy/pull/81#discussion_r1145421649

Reply:
So do I understand correctly; it is to be expected that there is no blockhash header in the response for `getConsensusInfo`and its absence therefore causes this endpoint to fail?

Reply:
Yes, there is no block hash expected. The block hash is returned for queries where you can supply a block.

My comment there was to mean that you should take the best block from the response, not the header.